client/assets/js/main.js

```diff
@@ -21,4 +21,54 @@ const shortenURL = () => {
       document.getElementById('shortened-url-lists').innerHTML = data.message
   })
 }
+
 document.querySelector('.create-shorten-url').onclick = shortenURL
+
+renderurls(urls)
+
+//===================================
+const verifyURLs = (e) => {
+	if(e.target.elements.shortenedURL.value !== '' && e.target.elements.shortenedURL.value.length < 7){
+		console.log('Too short.')
+		return 1 //1 means the short URL is too short.
```

Comment:
Returning a 1 isn't terribly semantic, and obfuscates the code that you're building for future developers. It also makes this code less scalable, because now you're responsible for managing error codes whenever you add new form fields and features.

I'd consider creating an object at the beginning with two properties:

`
const verificationStatus = {
    isValid: true,
    errorMessages: []
}
`

This affords us some nice features:

1. Instead of going through lots of if/else if statements, we can write a series of independent checks (so no need for else). If x condition is met, then we take no action, otherwise we can change the verification object's status to false, and append an error message about why
2. When we return a result to our user, we can tell them EVERYTHING that's wrong with their input, instead of them changing one thing, resubmitting, and figuring out that something else is wrong
3. It puts all of our error codes in a single location, and we can see a clear connection between a condition that must be met, and an error message that's appended if the condition is not met. This means fewer comments in your code (because the code has become more semantic)